TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -0,0 +1,71 @@
+package org.firstinspires.ftc.teamcode;
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
+import com.qualcomm.robotcore.hardware.Gamepad;
+import com.qualcomm.robotcore.hardware.HardwareMap;
+
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
+public class Robot {
+    Telemetry telemetry;
+    String ROBOT_CAPTION = "Robot Status";
+    DriveTrain driveTrain;
+    LinearOpMode opMode;
+    Gamepad gamepad1 = new Gamepad();
+    Gamepad gamepad2 = new Gamepad();
+    HardwareInnov8Robot hwmap;
+
+    double startPosition = 0;
+    double endPosition = 0;
+
+    public Robot(Telemetry telemetry, HardwareMap hwmap, LinearOpMode opMode) {
+        this.opMode = opMode;
+        this.hwmap = new HardwareInnov8Robot(hwmap);
+        this.telemetry = telemetry;
+        driveTrain = new DriveTrain(this.telemetry, this.hwmap, this.opMode);
+        this.telemetry.addData(ROBOT_CAPTION, "ready to go");
+        this.telemetry.update();
+    }
+
+    public double getConfi() {
```

Comment:
I don't think this function does whatever we were originally intending

Reply:
Since I deleted the useVuforia method is it ok if I delete this too?

Reply:
Yea feel free to delete everything you think we don't need